pkg/operator/operator.go

```diff
@@ -56,25 +50,32 @@ func NewOpenShiftControllerManagerOperator(
 	kubeClient kubernetes.Interface,
 	recorder events.Recorder,
 ) *OpenShiftControllerManagerOperator {
+	proxyLister := proxyInformer.Lister()
+	q := workqueue.NewNamedRateLimitingQueue(
+		workqueue.DefaultControllerRateLimiter(), "KubeApiserverOperator")
+	w := workload.NewWorkload(targetImagePullSpec, proxyLister, kubeClient, recorder)
 	c := &OpenShiftControllerManagerOperator{
-		targetImagePullSpec:  targetImagePullSpec,
 		operatorConfigClient: operatorConfigClient,
-		proxyLister:          proxyInformer.Lister(),
 		kubeClient:           kubeClient,
-		queue:                workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "KubeApiserverOperator"),
+		queue:                q,
 		rateLimiter:          flowcontrol.NewTokenBucketRateLimiter(0.05 /*3 per minute*/, 4),
-		recorder:             recorder,
+		workload:             w,
 	}
 
 	operatorConfigInformer.Informer().AddEventHandler(c.eventHandler())
 	proxyInformer.Informer().AddEventHandler(c.eventHandler())
-	kubeInformersForOpenshiftControllerManager.Core().V1().ConfigMaps().Informer().AddEventHandler(c.eventHandler())
-	kubeInformersForOpenshiftControllerManager.Core().V1().ServiceAccounts().Informer().AddEventHandler(c.eventHandler())
-	kubeInformersForOpenshiftControllerManager.Core().V1().Services().Informer().AddEventHandler(c.eventHandler())
-	kubeInformersForOpenshiftControllerManager.Apps().V1().Deployments().Informer().AddEventHandler(c.eventHandler())
+	kubeInformersForOpenshiftControllerManager.Core().V1().ConfigMaps().
```

Comment:
We don't usually split lines based on a certain character count, I think it makes it harder  to read (we went through this with the image registry/image registry operator also, but I think that was before your time)

Reply:
Okay, I understand that can be a polemic subject. I'm revering the changes :-)